### ISSUE TITLE
whether supports promise feature?

### ISSUE THREAD

**Issue by tiny09** [bda-research/node-crawler#332]
For example, I would like to do something after two different queue  invoked.

**Comment by mike442144** [comment]
Nope. but I believe you could queue in a single one. Could you show me your code?

**Comment by jo0ger** [comment]
@tiny09 
I think this article is helpful to you
https://jooger.me/article/5e342110ce986d002897d0c2

**Comment by mike442144** [comment]
@jo0ger Thanks for your article, I read it but I'm confused because you use the same function to handle the response of two seeds, why don't you just queue two urls as an array?
